packages/shopping/src/datasources/recommender-grpc.datasource.ts

```diff
@@ -7,7 +7,12 @@ import {inject} from '@loopback/core';
 import {juggler, AnyObject} from '@loopback/repository';
 import path from 'path';
 
-const config = require('./recommender-grpc.datasource.config.json');
+const config = {
+  name: 'recommender_grpc',
+  connector: 'loopback-connector-grpc',
+  url: '127.0.0.1:50051',
```

Comment:
The old config has port as 50000, I think it breaks the test.

`"url": "127.0.0.1:50000",`

Reply:
aha! updated accordingly